app/scripts/views/mixins/checkbox-mixin.js

```diff
@@ -17,6 +18,12 @@ define([
       'change input[type=checkbox]': 'logCheckboxChange'
     },
 
+    afterRender: function () {
+      this.$el.find('.fxa-checkbox').get().forEach(function (el) {
```

Comment:
What's the `get()` do here, I thought a `get` fetched a single specific element?